### ISSUE TITLE
Update for Hylang 0.27

### ISSUE THREAD

**Issue by wrobell** [pyx/hymn#24]
Hymn uses, it seems, out-of-date import syntax.

Trying first example in README file
```
$ hy
Hy 0.27.0 using CPython(main) 3.11.3 on Linux
=> (import [hymn.types.continuation [cont-m call-cc]])
Traceback (most recent call last):
  File "stdin-5736eb665650b2b8be3972494d6047763226f282", line 1
    (import [hymn.types.continuation [cont-m call-cc]])
            ^
hy.errors.HySyntaxError: parse error for pattern macro 'import': got unexpected token: hy.models.List([
  hy.models.Expression([
    hy.models.Symbol('.'),
    hy.models.Symbol('hymn'),
    hy.models.Symbol('types'),
    hy.models.Symbol('continuation')]),
  hy.models.List([
    hy.models.Symbol('cont-m'),
    hy.models.Symbol('call-cc')])]), expected: end of input
```

If I correct the example, then the same error, just different place
```
=> (import hymn.types.continuation [cont-m call-cc])
Traceback (most recent call last):
  File "stdin-41e774beeb13d04cc6be03e6444d5a84b2711690", line 1, in <module>
    (import hymn.types.continuation [cont-m call-cc])
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/wrobell/.local/lib/python3.11/site-packages/hymn/types/continuation.hy", line 7
    [hymn.types.monad [Monad]])
    ^
hy.errors.HySyntaxError: parse error for pattern macro 'import': got unexpected token: hy.models.List([
  hy.models.Expression([
    hy.models.Symbol('.'),
    hy.models.Symbol('hymn'),
    hy.models.Symbol('types'),
    hy.models.Symbol('monad')]),
  hy.models.List([
    hy.models.Symbol('Monad')])]), expected: end of input
```

**Comment by wrobell** [comment]
See also https://github.com/pyx/hymn/pull/25.

**Comment by pyx** [comment]
Thank you. #25 merged.

**Comment by pyx** [comment]
I released 1.0 supporting hy 0.27 now.
